Source/WebKit/Platform/spi/Cocoa/ExtensionKitSPI.h

```diff
@@ -111,4 +116,30 @@ NS_ASSUME_NONNULL_END
 @property (nonatomic, readonly) NSString *mediaEnvironment;
 @end
 
+@interface _SEHostingHandle: NSObject
+-(instancetype)initFromXPCRepresentation:(xpc_object_t)xpcRepresentation;
+-(xpc_object_t)xpcRepresentation;
+@end
+
+@interface _SEHostable: NSObject
++(_SEHostable*)createHostableWithOptions:(NSDictionary*)dict error:(NSError**)error;
```

Comment:
ObjC style isn't to use `create`, it would be just `hostableWithOptions:` and return an autoreleased object.